src/elements/content-picker/Footer.js

```diff
@@ -96,10 +97,11 @@ const Footer = ({
                                 <IconClose height={16} width={16} />
                             </Button>
                         </Tooltip>
-                        <Tooltip isDisabled={!selectedCount} text={chooseButtonLabel || chooseMessage}>
+                        <Tooltip isDisabled={isChooseButtonDisabled} text={chooseButtonLabel || chooseMessage}>
                             <PrimaryButton
                                 aria-label={chooseMessage}
-                                isDisabled={!selectedCount}
+                                disabled={isChooseButtonDisabled} // sets disabled attribute
+                                isDisabled={isChooseButtonDisabled} // used in Button component
```

Comment:
lol these comments aren't really necessary either

Reply:
I think we do need them, because otherwise ppl will be confused about what the difference is

Reply:
yeah you're right